presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedTestBase.java

```diff
@@ -2311,6 +2311,7 @@ public void testInformationSchemaQueries()
         assertQuery("SELECT table_name FROM iceberg.information_schema.tables WHERE table_schema ='test_schema2'", "VALUES 'iceberg_t3', 'iceberg_t4'");
         //query on non-existing schema
         assertQueryReturnsEmptyResult("SELECT table_name FROM iceberg.information_schema.tables WHERE table_schema = 'NON_EXISTING_SCHEMA'");
+        assertQueryReturnsEmptyResult("SELECT table_name FROM iceberg.information_schema.tables WHERE table_schema = 'non_existing_schema'");
```

Comment:
Isn't the other query above this line supposed to check this already. What's the difference?

Reply:
Makes sense, thanks for explaining